documentation/feature/maptools.md

```diff
@@ -6,21 +6,31 @@ group: feature
 parent: wiki
 ---
 
-## Overview
+## 1. Overview
 
-### Sub-feature 1
-Short description of sub-feature 1.
+### 1.1 Line drawing
+This adds the possibility to draw accurate lines.
 
-### Sub-feature 2
-Short description of sub-feature 2.
+### 1.2 Map tools
+This adds map tools that can be used to measure distances between two points on the map or bearings.
 
+### 1.3 GPS on map
+If you are equipped with a vanilla GPS it will be shown on the map. (You don't need the map tools in your inventory for this.)
 
-## Usage
+## 2. Usage
 
-Short overview of how to use the feature, e.g. menu options, key bindings, 
-instructions. May not apply to all modules.
+### 2.1 Using map tools
+- For this you need to have `Map Tools`.
+- Open the map <kbd> M </kbd> (by default).
+- Press the self interaction key <kbd> ctrl+left windows </kbd> (by default).
+- Select `Map tools`.
```

Comment:
You should probably add that the Roamer can be dragged with the LMB and rotated using Crtl+LMB.